mirgecom/initializers.py

```diff
@@ -370,6 +465,76 @@ def exact_rhs(self, discr, q, t=0.0):
         return flat_obj_array(massrhs, energyrhs, momrhs)
 
 
+class AcousticPulse:
+    r"""
+    Create an N-dimensional Gaussian acoustic pulse.
+
+    The Gaussian pulse is defined by:
+
+    .. math::
+
+        {\rho}E(\vec{r}) = {\rho}E + amplitude * G(\vec{r})\\
```

Comment:
`amplitude` is not useful/valid TeX. TeX typesets this as "variable a times variable m times variable p ...".